azurelinuxagent/common/utils/networkutil.py

```diff
@@ -175,6 +189,13 @@ def get_firewalld_accept_command(command, destination, uid, wait=""):
                                                                 uid))
         return cmd
 
+    @staticmethod
+    def get_firewalld_accept_command_nonroot_tcp(command, destination, wait=""):
```

Comment:
This is not being used anywhere currently, is that part of the 2nd PR where you'll figure out where/how to add it?

Reply:
You are right , firewalld changes are also included now